### PR TITLE
Add configuration scanned count in the report summary.

### PR DESCRIPTION
**Pull request by incfly**
This gave the users better clarity the scanner does do its job. `README.md` is also updated with new template format.

### CODE REVIEW

**Comment by jcchavezs** [comment]
At least new functions in `parser.go` require tests. Are we planning to add them?

**Comment by incfly** [comment]
Sure. We lack unit test. Let me add them.

File: README.md [hunk 1]
```diff
@@ -57,9 +52,43 @@ CVE Report
...
-- failed to find cluster role and role bindings to control istio gateway creation
+We scanned 0 security configurations, and 0 networking configurations.
+
+❌ failed to find cluster role and role bindings to control istio gateway creation
 ```
 
 ### Config Scanning
 
-TODO(incfly): more on the config issue pattern detection.
+Now we try to apply some configuration to see how the analyer can help detecting the potential security issues.
```
Comment (jcchavezs) [inline README.md:62]:
```suggestion
Now we try to apply some configuration to see how the analyzer can help detecting the potential security issues.
```

**Comment by incfly** [comment]
unit test added. @jcchavezs

**Review by jcchavezs — COMMENTED** [review]
Nice work.

**Pull request merged**